tests/test_version_check.py

```diff
@@ -0,0 +1,158 @@
+from unittest.mock import MagicMock, patch
+
+import pytest
+import requests
+
+import flow360.version_check as vc
+
+
+def test_get_supported_server_versions():
+    # Prepare mock data
+    mock_response = MagicMock()
+    mock_response.__iter__.return_value = iter([{"version": "1.0.0"}, {"version": "2.0.3b5"}])
+    # Mock the http_util.Http.get method
+    with patch("flow360.version_check.http_util.Http.get") as mock_get:
```

Comment:
I like this mock webapi testing very much! Can you integrate its code with https://github.com/flexcompute/Flow360/blob/develop/tests/mock_server.py